### ISSUE TITLE
error! [0]

### ISSUE THREAD

**Issue by gitalexa** [lucasasselli/garmin-podcasts#31]
- on trying to download episodes this error appears on my 955

**Comment by lucasasselli** [comment]
This is a CIQ "UNKNOWN_ERROR". No idea what could be the issue. Does it always happens?

**Comment by gitalexa** [comment]
yes always on trying to download. I can send logs, but i don't know how to create some.

**Comment by RoXXoR** [comment]
Having the same issue with my new FR955. It seems to be related with podcasts which are only accessable via https.
Checked with two podcast episode downloads at the same time (one http the other one https) and only the http was synchronized.
Might be related to this issue: https://www.playrun.app/blog/sync-issue
